<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NlSimpleCreate</class>
 <widget class="QDialog" name="NlSimpleCreate">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QTabWidget" name="tabWidget">
         <property name="currentIndex">
          <number>1</number>
         </property>
         <widget class="TQtWidget" name="cgmsTab">
          <attribute name="title">
           <string>Tab 1</string>
          </attribute>
         </widget>
         <widget class="TQtWidget" name="bolusTab">
          <attribute name="title">
           <string>Tab 2</string>
          </attribute>
         </widget>
         <widget class="TQtWidget" name="carbTab">
          <attribute name="title">
           <string>Page</string>
          </attribute>
         </widget>
         <widget class="TQtWidget" name="meterTab">
          <attribute name="title">
           <string>Page</string>
          </attribute>
         </widget>
         <widget class="TQtWidget" name="customTab">
          <attribute name="title">
           <string>Page</string>
          </attribute>
         </widget>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QPushButton" name="m_cgmsButton">
             <property name="text">
              <string>Add CGMS Data</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="m_bolusButton">
             <property name="text">
              <string>Add Bolus Data</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="m_carbButton">
             <property name="text">
              <string>Add Carb Data</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="m_meterButton">
             <property name="text">
              <string>Add Meter Data</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="m_cutomData">
             <property name="text">
              <string>Add Custom Data</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="m_excersizeButton">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>Add Exerc. Data</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="sizeConstraint">
            <enum>QLayout::SetMaximumSize</enum>
           </property>
           <item>
            <widget class="QLabel" name="label_5">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>Zoom X-Axis</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignBottom|Qt::AlignHCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QPushButton" name="zoomInButton">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>+</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="zoomOutButton">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>-</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Discard Data Before</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDateTimeEdit" name="m_startDateEntry"/>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Preferred</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Or After</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDateTimeEdit" name="m_endDateEntry"/>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Basal Rate</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="m_basalInsulinAmount"/>
         </item>
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>U/hour</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="m_openExisting">
         <property name="text">
          <string>Open Existing</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="m_cancelButton">
         <property name="text">
          <string>Cancel</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="m_createButton">
         <property name="text">
          <string>Create</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TQtWidget</class>
   <extends>QWidget</extends>
   <header>TQtWidget.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>m_excersizeButton</tabstop>
  <tabstop>m_createButton</tabstop>
  <tabstop>m_cgmsButton</tabstop>
  <tabstop>m_bolusButton</tabstop>
  <tabstop>m_carbButton</tabstop>
  <tabstop>m_meterButton</tabstop>
  <tabstop>m_cutomData</tabstop>
  <tabstop>m_startDateEntry</tabstop>
  <tabstop>m_endDateEntry</tabstop>
  <tabstop>m_basalInsulinAmount</tabstop>
  <tabstop>zoomInButton</tabstop>
  <tabstop>zoomOutButton</tabstop>
  <tabstop>m_openExisting</tabstop>
  <tabstop>m_cancelButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>m_cgmsButton</sender>
   <signal>clicked()</signal>
   <receiver>NlSimpleCreate</receiver>
   <slot>addCgmsData()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>548</x>
     <y>35</y>
    </hint>
    <hint type="destinationlabel">
     <x>319</x>
     <y>239</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_bolusButton</sender>
   <signal>clicked()</signal>
   <receiver>NlSimpleCreate</receiver>
   <slot>addBolusData()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>548</x>
     <y>69</y>
    </hint>
    <hint type="destinationlabel">
     <x>319</x>
     <y>239</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_carbButton</sender>
   <signal>clicked()</signal>
   <receiver>NlSimpleCreate</receiver>
   <slot>addCarbData()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>548</x>
     <y>103</y>
    </hint>
    <hint type="destinationlabel">
     <x>319</x>
     <y>239</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_meterButton</sender>
   <signal>clicked()</signal>
   <receiver>NlSimpleCreate</receiver>
   <slot>addMeterData()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>548</x>
     <y>137</y>
    </hint>
    <hint type="destinationlabel">
     <x>319</x>
     <y>239</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_cutomData</sender>
   <signal>clicked()</signal>
   <receiver>NlSimpleCreate</receiver>
   <slot>addCustomData()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>548</x>
     <y>171</y>
    </hint>
    <hint type="destinationlabel">
     <x>319</x>
     <y>239</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>zoomInButton</sender>
   <signal>clicked()</signal>
   <receiver>NlSimpleCreate</receiver>
   <slot>zoomInX()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>515</x>
     <y>359</y>
    </hint>
    <hint type="destinationlabel">
     <x>319</x>
     <y>239</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>zoomOutButton</sender>
   <signal>clicked()</signal>
   <receiver>NlSimpleCreate</receiver>
   <slot>zoomOutX()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>581</x>
     <y>359</y>
    </hint>
    <hint type="destinationlabel">
     <x>319</x>
     <y>239</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_startDateEntry</sender>
   <signal>editingFinished()</signal>
   <receiver>NlSimpleCreate</receiver>
   <slot>handleTimeLimitButton()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>226</x>
     <y>403</y>
    </hint>
    <hint type="destinationlabel">
     <x>319</x>
     <y>239</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_endDateEntry</sender>
   <signal>editingFinished()</signal>
   <receiver>NlSimpleCreate</receiver>
   <slot>handleTimeLimitButton()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>486</x>
     <y>403</y>
    </hint>
    <hint type="destinationlabel">
     <x>319</x>
     <y>239</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_basalInsulinAmount</sender>
   <signal>editingFinished()</signal>
   <receiver>NlSimpleCreate</receiver>
   <slot>enableCreateButton()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>117</x>
     <y>445</y>
    </hint>
    <hint type="destinationlabel">
     <x>319</x>
     <y>239</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_openExisting</sender>
   <signal>clicked()</signal>
   <receiver>NlSimpleCreate</receiver>
   <slot>openDefinedModel()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>394</x>
     <y>446</y>
    </hint>
    <hint type="destinationlabel">
     <x>319</x>
     <y>239</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>NlSimpleCreate</receiver>
   <slot>cancel()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>502</x>
     <y>446</y>
    </hint>
    <hint type="destinationlabel">
     <x>319</x>
     <y>239</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_createButton</sender>
   <signal>clicked()</signal>
   <receiver>NlSimpleCreate</receiver>
   <slot>constructModel()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>585</x>
     <y>446</y>
    </hint>
    <hint type="destinationlabel">
     <x>319</x>
     <y>239</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>addCgmsData()</slot>
  <slot>addBolusData()</slot>
  <slot>addCarbData()</slot>
  <slot>addMeterData()</slot>
  <slot>addCustomData()</slot>
  <slot>constructModel()</slot>
  <slot>openDefinedModel()</slot>
  <slot>cancel()</slot>
  <slot>handleTimeLimitButton()</slot>
  <slot>enableCreateButton()</slot>
  <slot>zoomInX()</slot>
  <slot>zoomOutX()</slot>
 </slots>
</ui>
